### PR TITLE
Added installation guide for linux deploy

### LINKED ISSUE #61: Bug: Correct README to show correct directions for Linux, and how to set up virtual environment
**Description:**
The directions to run the project on the [README.md](README.md) file show a really simplistic set of steps for working with Python to run the app. These can be incorrect due to operating systems, multiple Python installs (2.x and 3.x), or some other factors. Some want to use virtual environments instead.

This issue should make more specific directions for operating systems as well as include virtual environments.

**Requirements:**
* Should have valid directions to run the app using Python 3 (probably with the `python3` command instead)
* Should offer a way to do virtual environments (`pyenv` or `venv` perhaps)
* Should include directions a bit more specific to Linux.
* Does _not_ need to tell how to install Python, but should link to places to install it (or just show something like `brew install python` or `apt-get install python3` or `choco install  python3` or other variations)
* _Does_ need to assume a user is not familiar with the install process of this app and try to make it as clear and friendly as possible

**Claiming This Issue:**
* Please leave a comment below indicating your interest. I can then assign it to you.
* If I don't see an update (comment, code, or PR), then I'll take your name off.
* You would always be welcome to reclaim it again.
* If you can't work on it, either leave a comment or remove yourself from the issue.
* If you have questions, please let me know!

### PR DESCRIPTION
**Pull request by jlolzzz**
Added links for installation guides covering python3, and pip3 in most linux distros. Added quick deploy steps for pyenv and venv, and links or commands for installation of both. Could not find a direct installation module for venv in yum and pacman pachage managers, it may be handled through python installation. Otherwise, it should cover enough resources for any experience level to be able to deploy. If I missed something please let me know. Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,50 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
...
+
+django installation
+`pip3 install django`
+
+Recommended for use with python 3.6 and later
+Install venv for virtual environment (optional)
+`sudo apt install -y python3-venv` - Debian
+Full python3 and venv setup [centOS](https://www.i2tutorials.com/how-to-install-python-set-up-programming-environment-on-centos/)
+
+Recommended for use with python 3.3 and 3.5 deprecated in python 3.6
```
Comment (geekygirlsarah) [inline README.md]: Since 3.8 is the latest version, we can probably just cut this section entirely out.

File: README.md [hunk 2]
```diff
@@ -19,9 +19,50 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
...
+1. Clone the project (`git clone https://github.com/codethesaurus/codethesaur.us.git`)
+1. Switch into to directory `cd codethesaur.us`
+1. Use directory as virtual environment `python3 -m venv codethesaur.us`
+1. Activate the directory `source codethesaur.us/activate`
+1. Run `pip install -r requirements.txt`
+1. Then run `python manage.py runserver`
+1. In your browser, visit [http://127.0.0.1:8000/](http://127.0.0.1:8000/) or [http://localhost:8000/](http://localhost:8000/)
+1. Press CTRL+C in the terminal to stop the server.
+
+**Run using pyenv**
```
Comment (geekygirlsarah) [inline README.md]: We can probably remove this section too due to removing the above depreciated section.

File: README.md [hunk 3]
```diff
@@ -19,9 +19,50 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 
 ## Cloning and running it locally
 
+**Linux deployment:**
+
+Optional installation guides:
```
Comment (geekygirlsarah) [inline README.md]: I'm not sure why this says "optional installation guides"... they basically need to install it all if it's not already installed. I'd say this line can go away.

For the lines below, can they be formatted as numbered lists like the steps above and below this? I think that'd make it a bit clearer.

**Comment by jlolzzz** [comment]
I will start working on those changes shortly. Including adding numbered format. To make sure I am clear, you would like the pyenv section removed due to deprecation after python 3.6.

**Comment by geekygirlsarah** [comment]
Yeah. If it's deprecated and `venv` is what most people use (which is what I gather), then let's just remove that @currywurstjake .

**Review by geekygirlsarah — APPROVED** [review]
I think this will work!

**Pull request merged**